IdokladSdk.UnitTests/Tests/Validation/Detailed/Model/ModelWithNullableHasNoIdentificationNumber.cs

```diff
@@ -0,0 +1,12 @@
+﻿using IdokladSdk.Validation.Attributes;
+
+namespace IdokladSdk.UnitTests.Tests.Validation.Detailed.Model
+{
+    public class ModelWithNullableHasNoIdentificationNumber
```

Comment:
Mně není na první pohled jasný, proč je jeden model s nullable a druhý ne, když by mohl být prostě jeden model a to s nullable a tam bys mohl dát i null i hodnotu... Pochopil jsem, že ten s nullable je pro testování patche. Pak bych ho pojmenoval spíš podle toho patche, ať je jeho účel jasnější.

Reply:
Chápu, taky jsem váhal. Nakonec jsem zachoval již existující strukturu pojmenování.